messaging/src/main/java/org/axonframework/messaging/RemoteExceptionDescription.java

```diff
@@ -61,12 +69,34 @@ public static RemoteExceptionDescription describing(Throwable exception) {
      *
      * @param descriptions a {@link List} of {@link String}s, each describing a single "cause" on the remote end
      */
-    @JsonCreator
-    @ConstructorProperties({"descriptions"})
     public RemoteExceptionDescription(@JsonProperty("descriptions") List<String> descriptions) {
+        this(descriptions, false);
+    }
+
+    /**
+     * Initialize a RemoteExceptionDescription with given {@code descriptions} describing the exception
+     * chain on the remote end of communication and an indicator whether the exception is
+     * considered persistent or transient.
```

Comment:
The wording is fine, but I think changing it in two sentences makes it a tad more readable:
```suggestion
     * chain on the remote end of communication. The {@code persistent} indicator defines whether the exception is
     * considered persistent or transient.
```